src/playermat/Playermat.ttslua

```diff
@@ -394,7 +394,7 @@ function doUpkeep(_, clickedByColor, isRightClick)
       obj.call("doInUpkeep")
     end
 
-    if obj.type == "Card" and not inArea(self.positionToLocal(obj.getPosition()), INVESTIGATOR_AREA) then
+    if obj.type == "Card" and not inArea(self.positionToLocal(obj.getPosition()), INVESTIGATOR_AREA) and not obj.hasTag("Investigator") then
```

Comment:
Should removed the investigator area check now (it's surpassed by the tag check)

Reply:
```suggestion
    if obj.type == "Card" and not obj.hasTag("Investigator") then
```